packages/@webex/plugin-meetings/src/breakouts/breakout.ts

```diff
@@ -105,6 +106,34 @@ const Breakout = WebexPlugin.extend({
   parseRoster(locus) {
     this.members.locusParticipantsUpdate(locus);
   },
+
+  /**
+   * Broadcast message to this breakout session's participants
+   * @param {String} message
+   * @param {Object} options
+   * @returns {void}
+   */
+  broadcast(message, options) {
+    const roles = getBroadcastRoles(options);
+    const params = {
+      id: this.groupId,
+      recipientRoles: roles.length ? roles : undefined,
+      sessions: [
+        {
+          id: this.sessionId,
+        },
+      ],
+    };
+
+    return this.webex.request({
```

Comment:
issue: When there is nobody in a breakout session, the API errors with something like the following:
```json
{
    "errorCode": 201409036,
    "message": "Breakout message no valid recipients found",
    "errors": [
        {
            "errorCode": 201409036,
            "description": "Breakout message no valid recipients found"
        }
    ],
    "trackingId": "webex-js-sdk_36e33064-32fb-418e-a202-0bddd1e3513f_22"
}
```

We have two possibilities here
1) Catch the error and throw a suitable Error, one that can be caught by the consuming app.
2) Catch and hide this error here. i.e. if the error is a 429 and the error code is 201409036, catch and ignore.

Given it is a reasonably common scenario, and that the desktop clients also suppress this error, I suggest we do option 2.

Reply:
Got it. Catch the error 201409036 & 409 -> add a log and ignore